lib/actions/model.js

```diff
@@ -108,4 +108,11 @@ module.exports = {
       return validationErrors;
     };
   },
+
+  set(model, ...args) {
+    return () => {
+      model.set(...args);
```

Comment:
If we're not interacting with the store, let's not `dispatch` anything. Can this just be a callback passed into the React Component?